ethanschreur/kids-krafts
# Sellers can view information about the ordered products

**What does this PR do?**
- This pull request allows sellers to view information about the ordered products

**Description of the task to be completed** 
- Display the name and image of each product as well as how many of that product have been ordered, how many have been made, and how many still need to be made

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/products (log in if you have to)
- Fill out some products and give them subproducts
- Go to http://127.0.0.1:5000/orders
- Fill out some orders and add purchases to them
- Go to http://127.0.0.1:5000/dashboard
- Confirm that the information displayed about the ordered products is accurate
